package.json

```diff
@@ -27,9 +27,6 @@
     "desktop",
     "tablet"
   ],
-  "dependencies": {
-    "ua-parser-js": "^0.7.17"
-  },
```

Comment:
would you mind running `yarn remove ua-parser-js` so that the yarn.lock file also gets updated?

Reply:
Yes sry, did not know that. Most of the time i'm a backend developer. Thank you for the tip.